app/javascript/adminterface/lib/header_toggler.js

```diff
@@ -38,17 +38,19 @@ class HeaderToggler {
       this._add()
     }
 
-    this.element.addEventListener('click', function (e) {
-      e.preventDefault()
-      if ($container.classList.contains(_self.options.activeClass)) {
-        _self._remove()
-      } else {
-        _self._add()
-      }
-    })
+    this.element.addEventListener('click', (e) => this._clickCallBack(e, $container, _self))
 
     adminterface.addObserver(this.element, this, this.constructor.name)
   }
+
+  _clickCallBack (e, $container, _self) {
```

Comment:
Typo should be `_clickCallback`

Reply:
@ilunglee 
Sorry, so should I just edit the file directly on Github? Or should I edit it in my local machine, rebase, then force push to github? Like, what what you do in this situation?

Reply:
@dulerong,
You can make the changes to the branch on your local machine. And after you finished updating, you can: 
```
# stage the changes
$ git add .

# combine the changes with the latest commit in the history without editing the message
$ git commit --amend --no-edit
```
Then, you can force push to the remote`origin/jest_test_header_toggler` branch.

Cheers,